qiskit_experiments/framework/experiment_data.py

```diff
@@ -2268,6 +2223,18 @@ def load(
         expdata._retrieve_data()
         expdata._retrieve_analysis_results()
 
+        # Recreate artifacts
+        try:
+            if "artifact_files" in expdata.metadata:
+                for filename in expdata.metadata["artifact_files"]:
+                    if service.experiment_has_file(experiment_id, filename):
+                        artifact_file = service.file_download(experiment_id, filename)
+                        for artifact_string in zip_to_objs(artifact_file):
+                            artifact = json.loads(artifact_string, cls=cls._json_decoder)
```

Comment:
Same here. Can this be `zip_to_objs(..., decoder: Type[JSONDecoder]) -> list[Any]`?